frontend/ui/widget/opdsbrowser.lua

```diff
@@ -58,7 +58,37 @@ local OPDSBrowser = Menu:extend{
 }
 
 function OPDSBrowser:init()
-    self.item_table = self:genItemTableFromRoot(self.opds_servers)
+    local servers = G_reader_settings:readSetting("opds_servers") or {}
+    if next(servers) == nil then -- If there are no saved servers, add some defaults
+        servers = {
+          {
+	     title = "Project Gutenberg",
+	     url =
+	    "http://m.gutenberg.org/ebooks.opds/?format=opds",
+          },
+          {
+	     title = "Feedbooks",
+	     url =
+	    "http://www.feedbooks.com/publicdomain/catalog.atom",
+          },
+          {
+    	     title = "ManyBooks",
+    	     url = "http://manybooks.net/opds/index.php",
+          },
+          {
+    	     title = "Internet Archive",
+    	     url =
+    	    "http://bookserver.archive.org/catalog/",
+          },
+        }
+	for i in ipairs(servers) do
+            fields = {servers[i].title, servers[i].url}
+            self:addServerFromInput(fields) 
```

Comment:
If opds_servers setting was empty(nil), then we don't have to save the server list here, because next time it will just use the default list. If it wasn't empty, then we don't need to save it here either.